.travis.yml

```diff
@@ -1,28 +1,56 @@
 language: python
-
-python:
-  - "2.7"
-
 sudo: false
 cache: pip
 
+python:
+  # For branches <= 10.0, put `- "2.7.13"`
+  - "2.7.13"
```

Comment:
It seems there's a problem with this specific version. Put only 2.7 again to see

Reply:
Nope that's the line with virtualenv